contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
 
-## Cherry picks
+### Long-Term Stable (lts)
 
-We have a well-defined process for handling requests for changes to the canary release build or to the current production release build. These changes are known as "cherry picks".
+The **lts** release channel provides a previous **stable** build for four-week intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all publishers building experiences with AMP, but is provided so that those performing a QA cycle may do so less often. Individual pages can explicitly opt into **lts** (see [**lts** Readme](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md).
 
-### Cherry pick criteria
+Important: Publishers using the **lts** release channel should not use newly introduced features. Because of the four-week cycle, the **lts** release may be as much as six weeks behind the `HEAD` of [`ampproject/amphtml`](https://github.com/ampproject/amphtml). See the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release).
 
-**The bar for getting a cherry pick into canary or production is very high** because our goal is to produce high quality launches on a predictable schedule.
 
-**Keep in mind that performing a cherry pick requires a significant amount of work from you and the onduty person** and they can take a long time to process.
+## Determining if your change is in a release
 
-- In general only fixes for [P0 issues](https://github.com/ampproject/amphtml/blob/master/contributing/issue-priorities.md) (causing "an outage or a critical production issue") may be cherry picked. P0 issues are those that:
-  - cause privacy or security issues
-  - cause user data loss
-  - break existing AMP web pages in a significant way
-  - or would otherwise cause a significant harm to AMP's reputation if left unresolved
-- Regressions found in the canary that are not P0 _may_ be approved if they can be resolved with a rollback. Fixes other than rollbacks--no matter how simple they may seem--will not be approved because these can cause cascading problems and delay the release of canary to production for everyone.
+[_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to **experimantal**/**beta** testing to **stable**). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
```

Comment:
Suggestion: For clarity, remove parentheses around this text
> from the initial cut to **experimantal**/**beta** testing to **stable**

and rewrite sentence to
> ... are used to track the status of current and past releases from the initial cut, to testing via **experimantal**/**beta** channels, to eventual release via the **stable** channel.

Reply:
Done (with the addition of a semicolon for grammatical clarity)